Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -70,6 +70,7 @@ def print_filename(string, filename)
         print_filename(string, file) unless linked_libraries.empty?
         linked_libraries.each do |lib|
           puts " #{Tty.gray}-->#{Tty.reset} links to #{lib}"
+
```

Comment:
All these empty lines are so the places where `Tty` is used instead of a `Formatter` show up in the diff. I wasn't sure if these warrant a `Formatter`, or what to name it.